activities/MazeWeb.activity/css/activity.css

```diff
@@ -44,4 +44,22 @@
   background-color: #101010;
   overflow-x: hidden;
```

Comment:
Why not `overflox: hidden` instead of the separated properties?

Reply:
Can't find any good reason 👐